bot/exts/evergreen/minesweeper.py

```diff
@@ -159,9 +149,19 @@ async def start_command(self, ctx: commands.Context, bomb_chance: float = .2) ->
         except discord.errors.Forbidden:
             log.debug(f"{ctx.author.name} ({ctx.author.id}) has disabled DMs from server members")
             await ctx.send(f":x: {ctx.author.mention}, please enable DMs to play minesweeper.")
+            return
         else:
+            # Add game to list
+            board: GameBoard = self.generate_board(bomb_chance)
+            revealed_board: GameBoard = [["hidden"] * 10 for _ in range(10)]
             dm_msg = await ctx.author.send(f"Here's your board!\n{self.format_for_discord(revealed_board)}")
```

Comment:
As a side note, I probably wouldn't use the `else` clause here. Everything in the function after this point will also be only executed if no error is raised. Visually, it would make more sense to me to just end the `try-except` part with the `return`, and have everything after indented at the same level.